msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2024-05-07 00:16+0000\n"
"PO-Revision-Date: 2024-05-23 10:22\n"
"Last-Translator: \n"
"Language-Team: Hindi\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: hi\n"
"X-Crowdin-File: /lib/pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 3385\n"
"Language: hi_IN\n"

#: pylav/extension/bundled_node/manager.py:239
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture."
msgstr "आप प्रबंधित Lavalink नोड को किसी असमर्थित मशीन आर्किटेक्चर पर चलाने का प्रयास कर रहे हैं।"

#: pylav/extension/bundled_node/manager.py:466
msgid "The output of `{command_name_variable_do_not_translate}` was unexpected\n"
"{command_output_variable_do_not_translate}"
msgstr "का उत्पादन '{command_name_variable_do_not_translate}' अप्रत्याशित था\n"
"{command_output_variable_do_not_translate}"

#: pylav/extension/bundled_node/manager.py:489
msgid "The port {port_variable_do_not_translate} is already in use. Managed Lavalink startup has been aborted."
msgstr "बंदरगाह {port_variable_do_not_translate} पहले से ही उपयोग में है। प्रबंधित Lavalink स्टार्टअप को रद्द कर दिया गया है।"

#: pylav/extension/bundled_node/manager.py:494
msgid "Lavalink failed to start: {error_variable_do_not_translate}"
msgstr "Lavalink प्रारंभ करने में विफल रहा: {error_variable_do_not_translate}"

